onnxruntime/contrib_ops/rocm/diffusion/group_norm_ck.cuh

```diff
@@ -26,40 +26,44 @@ namespace rocm {
 
 using onnxruntime::rocm::CKDataTypeAdaptor;
 
-using Swish = ck::tensor_operation::element_wise::Swish;
+using Silu = ck::tensor_operation::element_wise::Swish;
```

Comment:
Since the naming disagree here, please add
> The SiLU function is also known as the swish function.

Reply:
The [updated paper](https://arxiv.org/abs/1710.05941v2) extends the definition of `Swish`: $$f(x)=x\cdot\sigma(\beta x)$$ but as quoted, PyTorch treats them as the same one.

Reply:
Comment are added.